tardis/io/tests/data/csvy_full.csvy

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
Is there a specific reason to have this at the beginning of the file?

Reply:
The astropy ecsv header begins with "---".  That being said we are definitely making some different choices already from the ecsv header design, so I would be open to not having the initial YAML_DELIMITER.

Reply:
I think its fine.